app/qml/components/MainPanelButton.qml

```diff
@@ -34,14 +34,20 @@ Rectangle {
     color: root.isHighlighted ? fontColor : backgroundColor
     opacity: root.faded ? 0.6 : 1
 
-    MouseArea {
+    Item {
         anchors.fill: parent
         enabled: root.enabled && handleClicks
-        onClicked: {
-            root.activated()
-        }
-        onPressAndHold: {
-            root.activatedOnHold()
+
+        TapHandler {
+            onPressedChanged: {
```

Comment:
Can we use `onTapped` instead? This way it would react even if one would press and release outside of the Item (if they wanted to cancel the action)

Reply:
If you say so. I just adopted one of the your previous fixes.